ipapython/ipaldap.py

```diff
@@ -76,6 +77,20 @@
 DIRMAN_DN = DN(('cn', 'directory manager'))
 
 
+if six.PY2:
+    # XXX silence python-ldap's BytesWarnings
```

Comment:
FYI, python-ldap 3.0.0b2 will make this a bit nicer and easier to fix, but it's going to take a couple of days.